xinference/client/restful/restful_client.py

```diff
@@ -171,6 +171,7 @@ class RESTfulImageModelHandle(RESTfulModelHandle):
     def text_to_image(
         self,
         prompt: str,
+        negative_prompt: str,
```

Comment:
This should be a optional parameter, not a required parameter. Otherwise it cannot be compatible with OpenAI SDK.
See https://platform.openai.com/docs/api-reference/images/create.

Reply:
fixed